tests/phpunit/integration/Core/Authentication/Setup_Test.php

```diff
@@ -206,7 +207,73 @@ function( $preempt, $args, $url ) use ( $context, &$proxy_server_requests, $has_
 		} catch ( WPDieException $exception ) {
 			$error = $has_credentials ? 'Test error message.' : 'test_error_code';
 			$this->assertStringContainsString(
-				sprintf( 'The request to the authentication proxy has failed with an error: %s', $error ),
+				sprintf(
+					'The request to the authentication proxy has failed with an error: %1$s %2$s.',
+					$error,
+					$oauth_proxy_failed_help_link
+				),
```

Comment:
Instead, just include the expected string directly - this is a more robust test, and clearer as to what the expectation is.

```suggestion
				sprintf(
					'The request to the authentication proxy has failed with an error: %1$s <a href="https://sitekit.withgoogle.com/support/?error_id=request_to_auth_proxy_failed" target="_blank">Get Help</a>.',
					$error
				),
```